packages/shared/src/models/bootc.ts

```diff
@@ -26,6 +26,7 @@ export interface BuildConfig {
   user?: BuildConfigUser[];
   filesystem?: BuildConfigFilesystem[];
   kernel?: BuildConfigKernel;
+  anacondaIsoInstallerModules?: BuildConfigAnacondaIsoInstallerModules[];
```

Comment:
It feels a little like the API is written to support the current UI as opposed to being functional. How about something like this, and letting the frontend build the lists instead of backend?
```
anacondaIsoInstallerModulesEnable?: string[];
anacondaIsoInstallerModulesDisable?: string[];
```